utils/validators.ts

```diff
@@ -314,7 +315,8 @@ export const parsePaybuttonTriggerPOSTRequest = function (params: PaybuttonTrigg
       buttonName: '',
       address: '',
       timestamp: 0,
-      opReturn: EMPTY_OP_RETURN
+      opReturn: EMPTY_OP_RETURN,
+      inputAddresses: ['']
```

Comment:
This looks wrong. Should be `[]`, that would be an empty input addresses list. `['']` is a list with one input address, namely `''`, which is unnecessary.